logging/tests/unit/handlers/transports/test_background_thread.py

```diff
@@ -264,6 +272,34 @@ def test__thread_main_batches(self):
         self.assertFalse(worker._cloud_logger._batch.commit_called)
         self.assertEqual(worker._queue.qsize(), 0)
 
+    def test__thread_main_max_latency(self):
```

Comment:
It might be useful to mock `time` here so that these tests are more deterministic.

Reply:
I don't know of an easy way to do this. If I mocked the `time` module, I would also have to mock `worker._queue.get`, which doesn't seem very clean. I'd personally be more comfortable with the test as-is.